gems/aws-eventstream/lib/aws-eventstream/bytes_buffer.rb

```diff
@@ -35,7 +37,7 @@ def readbyte
       end
 
       def write(bytes)
-        @data <<= bytes
+        @data += bytes
```

Comment:
Is this change related to freezing string literals, or just an extra change you wanted to make added in?

Reply:
yep, is related to freezing strings. but tests only break here if we add `frozen_string_literal` to:

```
rspec ./build_tools/aws-sdk-code-generator/spec/interfaces/client/event_stream_spec.rb
rspec ./build_tools/aws-sdk-code-generator/spec/protocols_spec.rb
rspec ./gems/aws-eventstream/spec/bytes_buffer_spec.rb
rspec ./gems/aws-eventstream/spec/decoder_spec.rb
rspec ./gems/aws-eventstream/spec/encoder_spec.rb
```

but I added in this PR just what needs to be changed to fix the problems I've found. I think that we can run

```
pragmater --add --comments "# frozen_string_literal: true" --includes "Gemfile" "Guardfile" "Rakefile" ".gemspec" "config.ru" "bin/**/*" "**/*.rake" "**/*.rb"
````

after merging this to ensure that everything stays compatible ;) [and that's how I found this kind of change]